image_transport_py/package.xml

```diff
@@ -0,0 +1,36 @@
+<package format="3">
+  <name>image_transport_py</name>
+
+  <version>5.0.3</version>
+
+  <description>Python API for image_transport</description>
+
+  <author>Tamas Foldi</author>
+
+  <maintainer email="alejandro@openrobotics.org">Alejandro Hernández</maintainer>
+  <maintainer email="john.dangelo@tailos.com">John D'Angelo</maintainer>
+
+  <license>BSD</license>
+
+  <url type="repository">https://github.com/ros-perception/image_common</url>
+  <url type="bugtracker">https://github.com/ros-perception/image_common/issues</url>
+
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_depend>ament_cmake_python</buildtool_depend>
+  <buildtool_depend>python_cmake_module</buildtool_depend>
+
+  <depend>image_transport</depend>
+  <depend>pybind11_vendor</depend>
```

Comment:
I don't know a lot about pybind11 - is this actually a depend or could it be a build depend? Seems like it would have a smaller footprint if this isn't actually a runtime dependency

Reply:
fair pont, it should be build depend only. Will change it